manifests/addons/gen.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+WD=$(dirname "$0")
+WD=$(cd "$WD"; pwd)
+
+set -eux
+
+# This script sets up the plain text rendered deployments for addons
+# See samples/addons/README.md for more information
+
+ADDONS="${WD}/../../samples/addons"
+DASHBOARDS="${WD}/../charts/istio-telemetry/grafana/dashboards"
+mkdir -p "${ADDONS}"
+
+# Set up Kiali. The upstream only has an operator, so we will deploy that
+# We set up anonymous authentication as this is for demos
+KIALI_SRC=$(mktemp -d)
+KIALI_VERSION=1.17.0
```

Comment:
@jmazzitelli any recommendation to use 1.17 to keep up with security patches vs 1.17.0?

Reply:
This is a git tag so it needs to be 1.17.0 (or 1.18.x soon), but we could make the image different than the version here I think

Reply:
Does it need to be a tag or can it be a branch name? We have vX.Y branch names.

Reply:
Note I just tried and github will package the tarball when referencing a branch as opposed to tag: e.g. https://github.com/kiali/kiali-operator/archive/v1.17.tar.gz so I think this will still do what you want while allowing this to keep up with z-stream patches.

Reply:
Oh nice, we can switch to that then